packages/pancake-liquidity-widgets/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@kyberswap/pancake-liquidity-widgets",
+  "license": "MIT",
+  "version": "0.0.1-rc",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@pancakeswap/sdk": "^5.8.2",
+    "@pancakeswap/v3-sdk": "^3.8.3",
+    "@popperjs/core": "^2.11.8",
+    "@reach/portal": "^0.18.0",
+    "d3": "^7.9.0",
+    "polished": "^4.3.1",
+    "typescript": "^5.2.2",
+    "viem": "2.13.3"
```

Comment:
Do you think this can be `peerDependencies`? 
also pancakeswap sdk and pancake v3 sdk?